### PR TITLE
kopia/repository/config/aws.go: Set session.Options profile from config

### LINKED ISSUE #6996: Non default s3 credential profiles do not work on Unified Repository Provider (kopia) - after #6598
Reopen of #6557 after #6598

Fix Re-implementation: #6995

- [ ] https://github.com/vmware-tanzu/velero/issues/6998

### PR DESCRIPTION
**Pull request by kaovilai**
Signed-off-by: Tiger Kaovilai <tkaovila@redhat.com>

Thank you for contributing to Velero!

# Please add a summary of your change
non default s3 credential profiles work again after #6598 

# Does your change fix a particular issue?

Fixes #6996

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: config/crd/v1/bases/velero.io_restores.yaml [hunk 1]
```diff
@@ -186,6 +186,12 @@ spec:
                                     - Continue
                                     - Fail
                                     type: string
+                                  waitForReady:
```
Comment (Lyndon-Li) [inline config/crd/v1/bases/velero.io_restores.yaml:189]: Looks like this is not relevant to the issue, what is the reason to make this change?
And as the principle to change a CRD, the yaml file is auto-generated, and should not be changed manually. We should change the `/pkg/api/velero/vx/*_types.go` only.

Comment (kaovilai) [reply config/crd/v1/bases/velero.io_restores.yaml:189]: CRDs were out of date from a prior PR.

Comment (kaovilai) [reply config/crd/v1/bases/velero.io_restores.yaml:189]: If someone would PR a `make update` only change and merge that I can rebase to make this PR this issue specific.
Another similar case
https://github.com/vmware-tanzu/velero/pull/7003/files#diff-49ea52a49c8a35bb42fdef90a38cc231783e10d8072be60b31cc884aa066cb74

Just didn't want to leave CI failing.

Comment (Lyndon-Li) [reply config/crd/v1/bases/velero.io_restores.yaml:189]: Got it, thanks.

**Review by Lyndon-Li — APPROVED** [review]

**Review by Lyndon-Li — APPROVED** [review]

**Review by sseago — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**